0.35.2 Release Notes (#3986)
astronomer/docs: software/manage-airflow-versions.md

:::

#### Runtime 11 (Airflow 2.9)

> ##### Restricted versions of Runtime 11

Author: ```suggestion
##### Yanked versions of Runtime 11
```

> You cannot create new Deployments or upgrade to the following versions of Runtime 11, which are `yanked`. These [restricted runtime versions](runtime-version-lifecycle-policy.mdx#restricted-runtime-versions) prevent you from upgrading to or creating a Deployment with a version that contains a known limitation or bug.

Reviewer: We aren't restricting but just warning users on yanked versions

> You cannot create new Deployments or upgrade to the following versions of Runtime 11, which are `yanked`. These [restricted runtime versions](runtime-version-lifecycle-policy.mdx#restricted-runtime-versions) prevent you from upgrading to or creating a Deployment with a version that contains a known limitation or bug.

Author: ```suggestion
Use caution if you create new Deployments or upgrade to the following versions of Runtime 11, which are `yanked`. These [Runtime versions](runtime-version-lifecycle-policy.mdx#restricted-runtime-versions) warn you about upgrading to or creating a Deployment with a version that contains a known limitation or bug.
```

- 11.0.0
- 11.1.0

##### Bug affecting users running a local Astro Runtime environment

In Airflow 2.9, a bug affecting custom actions in Airflow plugins ([#39421](https://github.com/apache/airflow/pull/39421)) prevented users from running the Astro Runtime environment locally for Astro Runtime versions 11.0.0, 11.1.0, and 11.2.0. Deployments running these versions on Astro are not affected.

#### Runtime 9 (Airflow 2.7)

> ##### Restricted versions of Runtime 9

Author: ```suggestion
##### Yanked versions of Runtime 9
```

> You cannot create new Deployments or upgrade to the following versions of Runtime 9, which are `yanked`. These [restricted runtime versions](runtime-version-lifecycle-policy.mdx#restricted-runtime-versions) prevent you from upgrading to or creating a Deployment with a version that contains a known limitation or bug.

Author: ```suggestion
Use caution if you create new Deployments or upgrade to the following versions of Runtime 9, which are `yanked`. These [Runtime versions](runtime-version-lifecycle-policy.mdx#restricted-runtime-versions) warn you about upgrading to or creating a Deployment with a version that contains a known limitation or bug.
```

#### Runtime 8 (Airflow 2.6)

> Astro Runtime version 8.0.0 is a restricted version of the Astro Runtime (`yanked`), which means you can't create Deployments on Astro with this runtime version. These [restricted runtime versions](runtime-version-lifecycle-policy.mdx#restricted-runtime-versions) prevent you from upgrading to or creating a Deployment with a version that contains a known limitation or bug.

Author: ```suggestion
Astro Runtime version 8.0.0 is a `yanked` version of the Astro Runtime, which means you are advised not to create Deployments with this runtime version. These [Runtime versions](runtime-version-lifecycle-policy.mdx#restricted-runtime-versions) warn you about upgrading to or creating a Deployment with a version that contains a known limitation or bug.
```
